fourTheorem/slic-starter
# Update deployment to work with latest serverless framework

- Use updated serverless path for recent versions
- Update welcome-service deps
- Regenerate package-lock for welcome-service
- Install latest NPM before audit
- Update serverless version
- Update bourne
- Update slic-tools version
- Use npm ci
- Move slic-tools dependencies to root
- Update package-locks for new slic-tools packaging
- Install root packages, use latest NPM
- Update path to aws-sdk mocked by aws-sdk-mock
- Fix up old pseudo parameter syntax
- Skip preflight check for eslint errors in frontend
- Rename .env.local to .env.development for new CRA resolution order
